src/App.jsx

```diff
@@ -14,6 +17,8 @@ const App = () => {
         <Routes>
           <Route path="/display/:key" element={ <DisplayComponent /> } />
           <Route path="/" element={ <ManagementComponent /> } />
+          <Route path="/" element={ <DiagnosticComponent /> } />
```

Comment:
These should be ... note the routing path ...

```javascript
<Route path="/diagnostic/:key" element={ <DiagnosticComponent /> } />
<Route path="/buzzer/:key" element={ <BuzzerComponent /> } />
```

And, they should be above the ManagementComponent route.